geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -546,7 +550,7 @@ private InternalDistributedSystem(ConnectionConfig config,
     quorumChecker = config.quorumChecker();
 
     ((DistributionConfigImpl) originalConfig).checkForDisallowedDefaults(); // throws
-                                                                            // IllegalStateEx
+    // IllegalStateEx
```

Comment:
I would delete or move the end-of-line comments.

Reply:
Fixed in an upcoming commit.